nvflare/fuel/f3/cellnet/cell.py

```diff
@@ -206,7 +285,7 @@ def _process_reply(self, future: StreamFuture):
         waiter.receiving_future = future
         waiter.in_receiving.set()
 
-    def register_request_cb(self, channel: str, topic: str, cb, *args, **kwargs):
+    def _register_request_cb(self, channel: str, topic: str, cb, *args, **kwargs):
```

Comment:
The register_request_cb() are being called several places. This renamed to "_register_request_cb()" will break them?